source/whats-new.txt

```diff
@@ -41,6 +42,97 @@ Learn what's new in:
 
 .. _c-upcoming-breaking-changes:
 
+.. _c-version-1.30.0:
+
+What's New in 1.30.0
+--------------------
+
+The following sections describe the changes in the 1.30.0 release of
+the {+driver-short+}.
+
+{+libbson+}
+~~~~~~~
+
+The 1.30.0 release of ``{+libbson+}`` includes the following changes:
+
+Deprecations
+````````````
+
+- Deprecates support for Debian 9 and Debian 10
+
+Improvements
+````````````
+
+- Improves the performance of BSON to JSON serialization for 
+  nested documents
+
+Fixes
+`````
+
+- Updates the truncated output of the ``bson_as_json_with_opts()`` function to no
+  longer split valid UTF-8 sequences
+
+- Updates generated JSON text in the following ways:
+
+  - Serializes empty arrays and documents with one space instead
+    of two spaces
+  - Serializes arrays that the driver omits because of the maximum recursion limit
+    to ``[...]`` instead of ``{...}``
+  - Applies the maximum recursion limit to legacy CodeWScope documents
+
+- Fixes an output parameter leaked after unsuccessful calls to the 
+  ``bson_append_array_builder_begin()`` function
+
+{+libmongoc+}
+~~~~~~~~~
+
+The 1.30.0 release of ``{+libmongoc+}`` includes the following changes:
+
+Deprecations
+````````````
+
+- Deprecates support for Debian 9 and Debian 10
+
+New Features
+````````````
+
+- Adds support for structured logging by using the
+  ``mongoc_client_set_structured_log_opts()`` and ``mongoc_client_pool_set_structured_log_opts()``
+  functions
+
+- Adds support for cache lifetime configuration of in-use encryption data
```

Comment:
```suggestion
- Adds support for cache lifetime configuration of In-Use Encryption data
```

Suggest using capitalization matching https://www.mongodb.com/docs/manual/core/security-in-use-encryption/. Consider linking to that page.